packages/nocodb/src/services/command-palette.service.ts

```diff
@@ -20,95 +20,73 @@ export class CommandPaletteService {
   async commandPalette(param: { body: any; user: UserType }) {
     const cmdData = [];
     try {
-      const { scope } = param.body;
+      const allBases = [];
 
-      if (scope === 'root') {
-        const bases = await Base.list({ user: param.user });
+      const bases = await BaseUser.getProjectsList(param.user.id, param);
 
-        for (const base of bases) {
-          cmdData.push({
-            id: `p-${base.id}`,
-            title: base.title,
-            icon: 'project',
-            iconColor: deserializeJSON(base.meta)?.iconColor,
-            section: 'Bases',
-            scopePayload: {
-              scope: `p-${base.id}`,
-              data: {
-                base_id: base.id,
-              },
-            },
-          });
-        }
-      } else if (scope.startsWith('p-')) {
-        const allBases = [];
-
-        const bases = await Base.list({ user: param.user });
+      allBases.push(...bases);
```

Comment:
Here no longer checking the palette scope, is it intentional?

cc: @mertmit

Reply:
we don't need scopes yet, looks good for now